src/components/elements/ErrorMessage.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react'
 import ErrorIcon from 'material-ui/svg-icons/alert/error'
+import { Card, CardHeader } from 'material-ui/Card'
 import { RetryButton } from 'components/elements/Buttons'
 
-const ErrorMessage = (message, actionForRetry) => <div>
-  <ErrorIcon />
-  <span>{message}</span>
-  { actionForRetry ? <RetryButton actionForRetry={actionForRetry} /> : null }
+const ErrorMessage = (message, actionForRetry, time = null, backgroundColor = null) => <div>
```

Comment:
We may want to validate that time is an instance of moment before we use it, and wrap it in a moment call if it's not. @maggiepint probably knows better what the best thing to do in these situations is.